modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+
+=== File Locking in the Web Interface
+
+ownCloud Server 10.5 comes with great enhancements for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out). 
+
+The feature builds on the xref:release_notes.html#webdav-locks[WebDAV Locks backend] which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
+
+To prevent files being locked infinitely, there is a mechanism that automatically expires locks after a certain time. The expiration time of locks can be configured via the "Manual File Locking" section in the _Admin -> Additional_ settings or using _occ_ commands:
+
+- _Default timeout for the locks if not specified (in seconds)_: Maximum lifetime of a lock set **via the web interface** (or by not specifying a timeout value when calling the WebDAV Locks API)
+`occ config:app:set core lock_timeout_default --value 1800`
+
+- _Maximum timeout for the locks (in seconds)_: Maximum lifetime of locks which is allowed to be set by calling the WebDAV Locks API
+`occ config:app:set core lock_timeout_max --value 86400`
+
+By default locks set in the web interface will expire after 30 minutes. The maximum lock time by default is one day.
+
+TIP: Please change the lock expiration settings according to your needs. Usually you will only need to change the default timeout as that applies to locks set in the web interface.
+
+TIP: The user-facing components in the web interface are disabled by default. Administrators can enable the feature by executing the following _occ_ command: `occ config:app:set files enable_lock_file_action --value yes`
+
+=== Changes to the ownCloud Marketplace
+
+The ownCloud Marketplace is opening up. With the release of Server 10.5, all apps available on the Marketplace (including ownCloud Enterprise apps) are available for download and installation via the Market app, respectively. This change facilitates the process of getting started with ownCloud Server and of evaluating Enterprise functionality. Additionally, it allows updates for all apps to be obtained from the ownCloud Marketplace and ensures running up-to-date versions of the extensions.
+
+=== Changes in Delivery
+
+In line with the changes to the Marketplace the deliverables for ownCloud Server have been unified. Previously there were different Tarball bundles, Docker images and Linux packages for the Community (bare minimum) and Enterprise (all supported apps) Editions.
```

Comment:
```suggestion
In line with the changes to the marketplace the deliverables for ownCloud server have been unified. Previously there were different tarball bundles, docker images and Linux packages for the community (bare minimum) and enterprise (all supported apps) editions.
```
I suggest to make most nouns lower case. It looks so german to me otherwise. 
The only exception in English should be uppercased nouns when they are coined terms.  E.g. the style guide could define an 'ownCloud Enterprise Server' with uppercase E and S. ... did not check that, sorry. 
And -- I am afraid -- this line is only one of many that needs review in this regard.

Reply:
Marketplace is a personal name, that one should be capitalized, the other ones should be in lower case

Reply:
`Marketplace`, `ownCloud Server`, `Community` and `Enterprise Edition` and `Docker` are proper names.
In essence, I agree that "tarball" should be lowercased.